### PR TITLE
Normalize path

### PR DESCRIPTION
**Pull request by hadley**

### CODE REVIEW

**Review by kevinushey — APPROVED** [review]

**Comment by kevinushey** [comment]
Let me see if I can reproduce the build failures on my Windows VM.

File: R/rtools.R [hunk 1]
```diff
@@ -127,11 +127,10 @@ renv_rtools_envvars_default <- function(root) {
 
   # add Rtools utilities to path
   bin <- normalizePath(file.path(root, "bin"), mustWork = FALSE)
-
   path <- paste(bin, Sys.getenv("PATH"), sep = .Platform$path.sep)
 
   # set BINPREF (note: trailing slash is required)
-  binpref <- file.path(renv_path_normalize(root), "mingw_$(WIN)/bin/")
+  binpref <- paste(renv_path_normalize(root), "mingw_$(WIN)/bin/", sep = "/")
```
Comment (hadley) [inline R/rtools.R:133]: FWIW I'm pretty sure `file.path()` uses the same separator on every platform.

Comment (kevinushey) [reply R/rtools.R:133]: `file.path()` drops trailing separators on Windows.

```
> file.path("a/b/")
[1] "a/b"
```

**Review by kevinushey — APPROVED** [review]
LGTM and tests seem happy; merge if you feel it's ready!

File: R/rtools.R (hunk 1 above)
Comment (hadley) [reply R/rtools.R:133]: Ooooooh that bastard!

**Comment by hadley** [comment]
Thanks for figuring this out! I'll revert the changes I made to unsuccessfully debug and merge it.

**Pull request merged**